Makefile

```diff
@@ -47,13 +48,13 @@ lint:
 	  --rule 'max-len: [off]' \
 	  -- test
 	$(REMEMBER_BOWER) $(shell pwd)
-	@echo 'Checking for missing link definitions...'
-	grep -o '\[[^]]*\]\[[^]]*\]' index.js \
-	| sort -u \
-	| sed -e 's:\[\(.*\)\]\[\]:\1:' \
-	      -e 's:\[.*\]\[\(.*\)\]:\1:' \
-	      -e '/0-9/d' \
-	| xargs -I '{}' sh -c "grep '^//[.] \[{}\]: ' index.js"
+	rm -f README.md
+	VERSION=0.0.0 make README.md
```

Comment:
We need to generate __README.md__ in order to check the link definitions in __index.js__. `VERSION` is not significant for this purpose, but I like to specify it as a reminder that it *is* required in order to generate a "valid" readme.